api/billingd/service/service.go

```diff
@@ -1075,7 +1091,15 @@ func (s *Service) reportCustomerUsage(ctx context.Context, cus *Customer) error
 			log.Warn("%s has invalid product key: %s", cus.Key, k)
 		}
 	}
-	s.analytics.Identify(cus.Key, cus.AccountType, false, "", summary)
+	if err := s.analytics.Identify(
+		ctx,
+		cus.Key,
+		cus.AccountType.Pb(),
+		analytics.WithEmail(cus.Email), // this wasn't being inclided before, do we want to?
```

Comment:
Just checking if we want to include email since it is available.

Reply:
should be safe either way